packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -254,35 +255,29 @@ const formatters = async (name, isTypescript) => {
     return
   }
 
-  const template = generateTemplate(
-    customOrDefaultTemplatePath({
-      side: 'web',
-      generator: 'scaffold',
-      templatePath: path.join('lib', 'formatters.tsx.template'),
-    }),
-    {
-      name,
-    }
-  )
+  const templateFileName = customOrDefaultTemplatePath({
+    side: 'web',
+    generator: 'scaffold',
+    templatePath: path.join('lib', 'formatters.tsx.template'),
+  })
 
-  const templateTest = generateTemplate(
-    customOrDefaultTemplatePath({
-      side: 'web',
-      generator: 'scaffold',
-      templatePath: path.join('lib', 'formatters.test.tsx.template'),
-    }),
-    {
-      name,
-    }
-  )
+  const template = generateTemplate(templateFileName, { name })
+
+  const templateTestFileName = customOrDefaultTemplatePath({
+    side: 'web',
+    generator: 'scaffold',
+    templatePath: path.join('lib', 'formatters.test.tsx.template'),
+  })
+
+  const templateTest = generateTemplate(templateTestFileName, { name })
 
   return {
     [outputPath]: isTypescript
-      ? template
+      ? prettify(templateFileName, template)
       : transformTSToJS(outputPath, template),
     [outputPathTest]: isTypescript
-      ? templateTest
-      : transformTSToJS(outputPathTest, templateTest),
+      ? prettify(templateFileName, template)
+      : transformTSToJS(templateTestFileName, templateTest),
```

Comment:
Based on the logic here, it reads as if only TS files get prettified. Maybe the transformTSToJS also prettifies, but have you confirmed?

Reply:
replied [here](https://github.com/redwoodjs/redwood/pull/7820#issuecomment-1471423007)